cypress/e2e/awx/resources/inventoryGroup.cy.ts

```diff
@@ -86,15 +86,90 @@ describe('Inventory Groups', () => {
       });
     });
 
-    it.skip('can run an ad-hoc command against a group', () => {
-      // cy.createInventoryHostGroup(organization).then((result) => {
-      // const { inventory, host, group } = result;
-      //1) Use the inventory created in beforeEach block, access the groups tab of that inventory
-      //2) Use the group, EE, and credential created in the beforeEach block- these resources are needed to run a command against a group
-      //3) Assert redirect to the job output screen
-      //4) Navigate to the details page of the job and assert the values there match what was entered in the Run Command Wizard
-      //5) Navigate back to the Inventory -> Jobs Tab to assert that the Run Command job shows up there
-      //});
+    it('can run an ad-hoc command against a group', () => {
+      cy.createInventoryHostGroup(organization).then((result) => {
+        const { inventory, group } = result;
+        //1) Use the inventory created in beforeEach block, access the groups tab of that inventory
+        cy.navigateTo('awx', 'inventories');
+        cy.filterTableBySingleSelect('name', inventory.name);
+        cy.clickTableRowLink('name', inventory.name, { disableFilter: true });
+        cy.verifyPageTitle(inventory.name);
```

Comment:
will you be able to reduce code duplication with navigation function?
from line 90 to 99 ?
there are a lot of instances of this lines with minor changes in them

Reply:
Do you have an example of this?

Reply:
This can be addressed in a future ticket